### ISSUE TITLE
Feature request: add option to delete mail directly in notification

### ISSUE THREAD

**Issue by digitalcraftsman** [k9mail/k-9#3232]
Hello K9-team,

with this issue I would like to propose the addition of a delete option for emails in notifications. Some emails (e.g. newsletters) are not relevant to me and can be moved directly into the trash bin.

Right now it's necessary to click on the notification to open K9 and then delete the email. It would be much faster if I could delete an email directly from the notification similar to the reply option.

This behavior becomes especially handy if you received multiple emails of this kind.

**Comment by internationils** [comment]
I would agree that this is useful, the only question is how to handle a notification that lists several emails...

**Comment by philipwhiuk** [comment]
There's already a delete option you can enable in settings.

Global Settings, Notifications, Show Delete Button

**Comment by digitalcraftsman** [comment]
Thanks for the tip @philipwhiuk
